docs/user-guide/api-mediation/configuration-extender-passtickets.md

```diff
@@ -1,46 +1,108 @@
+## Configuring Zowe to use PassTickets
 
-    - [ACF2](#acf2)
-    - [Top Secret](#top-secret)
-    - [RACF](#racf)
+Configuring Zowe to use PassTickets involves two processes:
+* Enabling PassTicket support for your ESM
+* Configuring security to allow the Zowe API Gateway to generate PassTickets for an API service
 
 ### Enabling PassTicket support
 
-The following steps outline the procedure for enabling PassTicket Support:
+The following steps outline the procedure for enabling PassTicket Support for your ESM:
 
-1. Follow the API service documentation that explains how to activate support for PassTickets.
+#### PassTicket enablement with ACF2
 
-  **Note:** PassTickets for the API service must have the replay protection switched off. 
-  
-  **Example:** `APPLDATA('NO REPLAY PROTECTION')
+1. Define PassTicket for application ID _applid_ without replay protection.
+```
+SET PROFILE(PTKTDATA) DIVISION(SSIGNON)
+INSERT <applid> SSKEY(<key-description>) MULT-USE
+F ACF2,REBUILD(PTK),CLASS(P)
+```
+2. Add the Zowe user to the defined application ID.
+```
+SET RESOURCE(APL)
+RECKEY <applid> ADD(UID(<zowe-user-id>) ALLOW)
+F ACF2,REBUILD(APL)
+```
 
-  The PassTickets are exchanged between Zowe API Gateway and the API service in a secure mainframe environment.
+#### PassTicket enablement with Top Secret
 
-2. Record the value of the APPLID of the API service.
-3. Enable the Zowe started task user ID to generate PassTickets for the API service. Grant `UPDATE` access to the Zowe started task by submitting commands in one of the three ESMs: ACF2, Top Secret, or RACF.
-4. Enable PassTicket support in the API Gateway for your API service.
+Define PassTicket for application ID _applid_ without replay protection.
 
-### Security configuration that allows the Zowe API Gateway to generate PassTickets for an API service
+```
+TSS ADDTO(NDT) PSTKAPPL(<applid>) SESSKEY(<key-description>) SIGNMULTI
+```
+
+#### PassTicket enablement with RACF
+
+1. Activate the `PTKTDATA` class, which encompasses all profiles containing PassTicket information. Execute the following command:
+```
+SETROPTS CLASSACT(PTKTDATA) RACLIST(PTKTDATA)
+```
+
+2. Specify the application ID requiring access through PassTicket for the ZOWE server with the following commands:
+```
+RDEFINE APPL <applid> UACC(READ)
+SETROPTS CLASSACT(APPL)
+SETROPTS GENERIC(PTKTDATA)
+```
+
+Replace _applid_ with a one to 8 character name designated for the application. 
+
+:::note 
+* This name is usually provided by the site security administrator.
+:::
+
+3. Define the profile for the application with the following command:
+```
+RDEFINE PTKTDATA <applid> SSIGNON(KEYMASKED(<key-description>))
+```
+* **key-description**  
+ Specifies the secured sign-on hexadecimal application key
+
+Replace with the application name defined previously.
+
+:::caution Important
+
+PassTickets for the API service must have the replay protection switched off. 
+
+```
+APPLDATA('NO REPLAY PROTECTION')
```

Comment:
This line is part of upper command - it does not make any sense to split it into separate unrunable snippet. What user should do with it? Copy for what?